docs/install.md

```diff
@@ -261,13 +261,15 @@ the board in order to see your OpenSK device on your system.
 Although flashing using a Segger JLink probe is the officially supported way,
 our tool, `deploy.py` also supports other methods:
 
-*   OpenOCD: `./deploy.py --board=nrf52840_dongle --opensk --programmer=openocd`
-*   pyOCD: `./deploy.py --board=nrf52840_dongle --opensk --programmer=pyocd`
-*   Nordic DFU: `./deploy.py --board=nrf52840_dongle --opensk
+*   OpenOCD: `./deploy.py --board=nrf52840_dongle_opensk --opensk
+    --programmer=openocd`
+*   pyOCD: `./deploy.py --board=nrf52840_dongle_opensk --opensk
+    --programmer=pyocd`
+*   Nordic DFU: `./deploy.py --board=nrf52840_dongle_opensk --opensk
```

Comment:
This one should be using another board, right?
Like `nrf52850_dfu_dongle` or something

Reply:
Right, that one is special indeed.